CHANGELOG.rst

```diff
@@ -1,5 +1,38 @@
 Changelog
 =========
+1.7.0 (2020-03-01)
+------------------
+MLflow 1.7.0 includes several major features and improvements:
+
+Model Registry APIs: breaking changes
```

Comment:
This is only REST APIs, right, not Python APIs? Most users won't see the REST ones at all so it would be good to say REST APIs. Also say what the impact is (which clients are compatible with new/old servers: do you have to upgrade both? if not, how long is compatibility going to be supported for?)

Reply:
Thanks @mateiz. Made the title more clear and added comments about clients maintaining backward compatibility at the API level and the user code does not need to change.